keyserver/README.md

```diff
@@ -0,0 +1,30 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+Generate new config file with proper format
+```
+go run utils/configen/mkconfig.go
+```
+
+Generate new key pair for signing
+``` 
+go run utils/keygen/mkkey.go -vrf -signing
```

Comment:
Make this two commands (one with signing and one with vrf flag) or change mkkey.go accordingly